ror.yml

```diff
@@ -0,0 +1,147 @@
+# Recipe for Ruby on Rails collections
+---
+rh-ror50:
+  name: Ruby on Rails 5.0
+  requires: [rh-ruby23, rh-nodejs4, rh-mongodb32]
+  # install: Copr - Settings - Build options - Click "Edit" on target
+  #   Packages
+  install:
+    - scl-utils-build
+    - rh-ruby23-scldevel
+    - rh-ror50-build
+  packages:
+    - rh-ror50
+    # Packages required by RSpec, Cucumber
+    - rubygem-rspec
+    - rubygem-rspec-core:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-rspec-support:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-diff-lcs:
+        macros:
+          _with_bootstrap: 1
```

Comment:
@pvalena ok for it's up to me.
This time I will not update Fedora and SCL to unify the flag style, as I do not have enough time to do it.

> What do you propose, then?

My current creating builder script with recipe is not only for Ruby, Rails but also for any recipe files.
So, if you mind to unify the flag style in my recipe, I thought that it did also mean to unify other recipe's style too.

So, finally it's time to merge this pull-request.
Thanks.